src/rbmc_data_sync_main.cpp

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: Apache-2.0
+
+#include <phosphor-logging/lg2.hpp>
+#include <sdbusplus/async/context.hpp>
+#include <sdbusplus/server/manager.hpp>
+#include <xyz/openbmc_project/DataSync/BMCData/common.hpp>
+
+int main()
+{
+    using BMCDataSync =
+        sdbusplus::common::xyz::openbmc_project::data_sync::BMCData;
+
+    sdbusplus::async::context ctx;
+    sdbusplus::server::manager_t objManager{ctx, BMCDataSync::namespace_path};
+
+    // clang-tidy currently mangles this into something unreadable
+    // NOLINTNEXTLINE
+    ctx.spawn([](sdbusplus::async::context& ctx) -> sdbusplus::async::task<> {
```

Comment:
can't we call it synchronously? before anything else this should happen right?

Reply:
Please refer to [this comment](https://github.com/ibm-openbmc/phosphor-data-sync/pull/1#discussion_r1877268267). This PR is specifically intended for reviewing commit 2.